src/test/java/com/crud/tasks/service/SimpleEmailServiceTest.java

```diff
@@ -0,0 +1,46 @@
+package com.crud.tasks.service;
+
+import com.crud.tasks.domain.Mail;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+import org.springframework.mail.SimpleMailMessage;
+import org.springframework.mail.javamail.JavaMailSender;
+
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+
+
+@RunWith(MockitoJUnitRunner.class)
+public class SimpleEmailServiceTest {
+
+    @InjectMocks
+    private SimpleEmailService simpleEmailService;
+
+    @Mock
+    private JavaMailSender javaMailSender;
+
+    @Test
+    public void shouldEmailSend() {
+        //Given
+        Mail mail = new Mail("test@test.com", "Test", "Test Message");
+
+        SimpleMailMessage mailMessage = new SimpleMailMessage();
+        mailMessage.setTo(mail.getMailTo());
+        mailMessage.setSubject(mail.getSubject());
+        mailMessage.setText(mail.getMessage());
+        if(mail.getToCc()!=null &&  mail.getToCc()!=""){
```

Comment:
w testach nie wciskaj logiki tak jak tutaj. po prostu jeśli chcesz ustawić CC to ustaw, a jeśli nie chcesz to nie. w teście powinieneś mieć sztywne dane